src/js/components/SidebarLabelsFilter.js

```diff
@@ -211,7 +211,7 @@ class SidebarLabelsFilters extends mixin(QueryParamsMixin) {
     }
 
     return (
-      <div className="side-list sidebar-filters hidden-medium hidden-small pod flush-top flush-left">
+      <div className="side-list sidebar-filters pod flush-top flush-left">
```

Comment:
I was able to remove these entirely because their parent, `ServiceSidebarFilters`, uses the appropriate class.